versions/2/changelog.yml

```diff
@@ -0,0 +1,323 @@
+##
+## This file documents changes in the package specification. It is NOT a package specification file.
+## Newer entries go at the bottom.
+##
+- version: 2.0.0-next
```

Comment:
Current approach keeps both majors in the same branch. This implies that releasing a minor for one major would require the release of new versions for all majors with changes. Releasing a new version will require to close all changelogs with changes, and push tags for all new versions mentioned in changelogs.

This makes me wonder a couple of things:
* AFAIK we are not using minors for anything, should we do something about this? Or should we reduce versioning to only majors?
  If we implement minors we could have more control on what is released for each major, but following current approach we would need to duplicate the spec and rules for each minor.
* Should we have one branch per major? This would complicate other things, we would need an entry point for `NewSpec`, that instantiates one spec per major from a different import path.

Reply:
With reducing versioning I mean to use `format_version: 2` in manifests. We would keep the versioning here in changelogs.

Reply:
We can completely freeze version 1 when 2 is released, but I would expect a period where version 2 is in development, there are several potentially breaking changes that will need some time to be implemented. At least during this time we are going to need some kind of synchronization between both versions.